style/servo/restyle_damage.rs

```diff
@@ -146,6 +147,13 @@ fn compute_damage(old: &ComputedValues, new: &ComputedValues) -> ServoRestyleDam
             old.get_effects().filter.0.is_empty() != new.get_effects().filter.0.is_empty()
     };
 
+    restyle_damage_rebuild_box(old, new) && rebuild_box_extra()
```

Comment:
Doesn't this need to be OR?

```suggestion
    restyle_damage_rebuild_box(old, new) || rebuild_box_extra()
```

And at this point you can inline `rebuild_box_extra()`